src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,101 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
-
+import classNames from 'classnames';
+import { PropTypes } from 'prop-types';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
+import { request } from '../../api';
+
+export const FindMovie = ({ movies, setMovies }) => {
+  const [value, setValue] = useState('');
+  const [movie, setMovie] = useState('');
+  const [error, setError] = useState('');
+  const handleFindMovie = title => (
+    request(title).then((res) => {
+      if (res.Response === 'True') {
+        setError('');
```

Comment:
```suggestion
        setError(false);
```
use boolean values to error